### DISCUSSION TITLE
examples of uses of textual?

### DISCUSSION THREAD

**Discussion by delulu3i4** [Textualize/textual#365] [Show and tell]
I know still have some items to finish, but think could be cool that in addition to like links to other projects using textual (eg like a gallery)

anybody have any examples want to share on this thread (even if just for fun, poc ,etc) ? 

For help of using textual, for just fun project, I was thinking would be cool to see how far can take in creating something similar to https://github.com/erroneousboat/slack-term/ which is terminal ui for slack. With textual and https://github.com/slackapi/python-slack-sdk seems like could quite cool (especially given the slack desktop app can take up quite a bit of memory and feel bit slow at times).

**Comment by mharig** [comment]
It's not my project, but I think it's  a rather usefull example:
[tiptop](https://github.com/nschloe/tiptop)

**Comment by hdb** [comment]
Here's an interface I built for TPB: https://github.com/hdb/baywatch

It's a search TUI that implements widget list refreshing, tab indexing, dynamic sidebars and a configuration editor - it makes use of a couple extensions folks have posted here already: [textual inputs](https://github.com/sirfuzzalot/textual-inputs) and [scrollable widget lists](https://github.com/Cvaniak/TextualListViewUnofficial)

**Comment by jeffwright13** [comment]
Here's my lame attempt at building a TUI around Pytest results (similar to what `pytest-html` provides, but automatically launched in the console after the tests are done). I am in the middle of rewriting the Textual TUI to more closely match what I've done in PyTermTk.

[pytest-tui](https://github.com/jeffwright13/pytest-tui)

**Comment by HiPeople21** [comment]
I built an editor with Textual.

[terminwind](https://github.com/HiPeople21/Terminwind)

**Comment by delulu3i4** [comment]
One project would be cool to see if any areas to take inspiration on is Go's TUI framework bubbletea on GitHub readme has some cool examples of tui's built with it and other component libraries https://github.com/charmbracelet/bubbletea

**Reply by jeffwright13** [reply]
> One project would be cool to see if any areas to take inspiration on is Go's TUI framework bubbletea on GitHub readme has some cool examples of tui's built with it and other component libraries https://github.com/charmbracelet/bubbletea

lol, i was just looking at bubbletea last night - looks cool too bad i can't program in go

**Comment by JoseAndresMedina** [comment]
I spent the last week playing around with Textual and although it's not installable through PyPI yet, I made a TicTacToe interface: 
[TextTacToe](https://github.com/JoseAndresMedina/TextTacToe)
